src/test/java/net/snowflake/client/jdbc/BindingDataLatestIT.java

```diff
@@ -43,4 +44,126 @@ public void testBindTimestampTZ() throws SQLException {
     assertThat("integer", resultSet.getInt(1), equalTo(123));
     assertThat("timestamp_tz", resultSet.getTimestamp(2), equalTo(ts));
   }
+
+  /**
+   * Test that stage binding and regular binding insert and return the same value for timestamp_ntz
+   *
+   * @throws SQLException
+   */
+  @Test
+  public void testTimestampBindingWithNTZType() throws SQLException {
+    try (Connection connection = getConnection()) {
+      TimeZone origTz = TimeZone.getDefault();
+      Statement statement = connection.createStatement();
+      statement.execute(
+          "create or replace table stageinsert(ind int, ltz0 timestamp_ltz, tz0 timestamp_tz, ntz0 timestamp_ntz)");
+      statement.execute(
+          "create or replace table regularinsert(ind int, ltz0 timestamp_ltz, tz0 timestamp_tz, ntz0 timestamp_ntz)");
+      statement.execute("alter session set CLIENT_TIMESTAMP_TYPE_MAPPING=TIMESTAMP_NTZ");
+      statement.execute("alter session set TIMEZONE='Asia/Tokyo'");
+      TimeZone.setDefault(TimeZone.getTimeZone("Asia/Tokyo"));
+      Timestamp currT = new Timestamp(System.currentTimeMillis());
+
+      // insert using stage binding
+      PreparedStatement prepStatement =
+          connection.prepareStatement("insert into stageinsert values (?,?,?,?)");
+      for (int i = 1; i <= 60000; i++) {
```

Comment:
Nit: we have an internal parameter called CLIENT_STAGE_ARRAY_BINDING_THRESHOLD that sets the stage binding threshold to the number of binds that can be used instead of reaching the threshold with a 60000 for loop. If the total number of binds is over the threshold, stage binding will be used (customers cannot see this). See PreparedStatementIT.java classes for more examples of usage. It will speed up the tests to use this param instead of the giant for loop.

```suggestion
      statement.execute("ALTER SESSION SET CLIENT_STAGE_ARRAY_BINDING_THRESHOLD = 1"); // enable stage bind
      PreparedStatement prepStatement =
          connection.prepareStatement("insert into stageinsert values (?,?,?,?)");
          prepStatement.setInt(1, 1);
        prepStatement.setTimestamp(2, currT);
        prepStatement.setTimestamp(3, currT);
        prepStatement.setTimestamp(4, currT);
        prepStatement.addBatch();
        prepStatement.executeBatch();
        statement.execute("ALTER SESSION SET CLIENT_STAGE_ARRAY_BINDING_THRESHOLD = 0"); // disable stage bind by setting threshold back to default

```

Reply:
@sfc-gh-mknister, I'm running JDBC tests and bunch of tests are failing because CLIENT_STAGE_ARRAY_BINDING_THRESHOLD can not be set (SQL compilation error:
invalid parameter 'CLIENT_STAGE_ARRAY_BINDING_THRESHOLD'). 
Is there something special I need to do to make sure these tests can run?